lib/fs.js

```diff
@@ -319,39 +329,46 @@ fs.existsSync = function(path) {
   }
 };
 
+const defaultReadFileOpts = Object.create(null, {
+  flag: { value: O_RDONLY, writable: true }
+});
```

Comment:
I'm not certain how much performance impact this may have, but because of the `Object.create(null, ...)`:

```js
const { O_RDONLY } = fs.constants;
const defaultReadFileOpts = Object.create(null, {
  flag: { value: O_RDONLY, writable: true }
});
console.log(%HasFastProperties(defaultReadFileOpts));
  // Prints false
```